src/GslibIO.jl

```diff
@@ -13,6 +13,17 @@ using GeoStatsBase
 # type aliases
 const Array2or3{T} = Union{AbstractArray{T,2},AbstractArray{T,3}}
 
+# legacy specification of the GSLIB format
+# (see documentation here: http://www.gslib.com/gslib_help/format.html)
+# Note that in this context variables can be
+# actual attributes as well as coordinates
+struct LegacySpec
+  header::String           # Content of the first line (documentation only)
+  nvars::Int               # An integer with the number of variables (first integer in the second line)
```

Comment:
It seems like we are not using this `nvars` field in the implementations as it is implicit in the field `varnames`. Can we remove it since it is redundant and not used anywhere later?